Dockerfile

```diff
@@ -1,17 +1,15 @@
-FROM ubuntu:latest
+FROM germanedge-docker.artifactory.new-solutions.com/edge-one/ge-ubuntu-generic:0.16.0
 
-ARG zookeper_version=3.5.8
-ARG consul_version=1.7.1
-ARG hashicorp_releases=https://releases.hashicorp.com
-ARG filebeat_version=7.5.0
-ARG consul_url=consul
+ARG zookeper_version=3.7.0
 
-ENV ZOOKEEPER_VERSION=$zookeper_version \
-    CONSUL_VERSION=$consul_version \
-    HASHICORP_RELEASES=$hashicorp_releases \
-    FILEBEAT_VERSION=$filebeat_version \
-    CONSUL_URL=$consul_url
+ENV ZOOKEEPER_VERSION=$zookeper_version
+ENV SERVICENAME=zookeeper
+ENV CONSUL_TAGS='"web","application","prometheus"'
+ENV CONSUL_META_SCRAPE_PATH="\/metrics"
+ENV CONSUL_META_SCRAPE_PORT="7071"
+ENV LOGSTASH_VERSION=$logstash_version
```

Comment:
Where LOGSTASH_VERSION env is used?

Reply:
that was a mistake, I'm going to remove it.

Reply:
Added env for read logs from other location